### ISSUE TITLE
[BUG] Not clear to user when registration page is unavailable/closed

### ISSUE THREAD

**Issue by Scarrien** [SFDO-Community-Sprints/Summit-Events-App#412]
Working with Midstate Technical College has identified some gaps in making sure staff users know when registration pages are available and closed. There was an event on Saturday where people were expecting to be registering for an event but were unable to because registration was actually closed, and some new/updated formula fields on the Instance can prevent this in the future.

- `Registration Close Date` should account for `Keep Registration Open During Event` on Event (for multi-day events), and needs help text
    - Help text: `Formula: The date that registration for this event closes based on values entered on the Event record (Event End Date, Close Event Days Before value)`
- `Open Registration` needs help text, should account for close time (needs to have same logic as the registration field display functionality, which it currently does not)
    - Help text: `Formula: Checked if the event is active, "Registration Close Date" is greater or equal to today, and there is still available capacity`
- `Registration Close Time` formula field should be made so it is clear when registration form will be unavailable
    - Help text: `Formula: The time registration ends based on the Event's "Keep Registration Open During Event" value`

```
IF(summit__Event__r.summit__Keep_Registration_Open_During_Event__c,
summit__Instance_End_Time__c,
summit__Instance_Start_Time__c)
```

**Comment by tcdahlberg** [comment]
Included in beta release 0.28.0.1

**Comment by tcdahlberg** [comment]
Included in production release 0.28.0.2
